lib/Invasion.js

```diff
@@ -52,7 +52,13 @@ class Invasion extends WorldstateObject {
      * The node where the invasion is taking place
      * @type {string}
      */
-    this.node = translator.node(data.Node, locale);
+    this.nodeKey = translator.node(data.Node, locale);
+
+    /**
+    * The node key where the invasion is taking place
+    * @type {string}
+    */
+    this.node = translator.node(data.Node);
```

Comment:
i think you swapped these
```suggestion
    this.node = translator.node(data.Node, locale);

    /**
    * The node key where the invasion is taking place
    * @type {string}
    */
    this.nodeKey = translator.node(data.Node);
```